sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1460,9 +1462,12 @@ def task_worker(self):
 
                         if self.port_dict[lport]['host_tx_ready'] != 'true' or \
                                 self.port_dict[lport]['admin_status'] != 'up':
-                           self.log_notice("{} Forcing Tx laser OFF".format(lport))
-                           # Force DataPath re-init
-                           api.tx_disable_channel(media_lanes_mask, True)
+                           if is_fast_reboot and self.check_datapath_state(api, host_lanes_mask, ['DataPathActivated']):
+                               self.log_notice("{} Skip re-init flow".format(lport))
```

Comment:
@stepanblyschak please add "Skip datapath re-init in fast-reboot" log

Reply:
@prgeor Done